CodeSniffer/Standards/PEAR/Sniffs/Commenting/FileCommentSniff.php

```diff
@@ -347,6 +347,12 @@ protected function processPackage(PHP_CodeSniffer_File $phpcsFile, array $tags)
             $newContent = preg_replace('/[^A-Za-z_]/', '', $newContent);
             $nameBits   = explode('_', $newContent);
             $firstBit   = array_shift($nameBits);
+            
+            if(!isset($firstBit{0})){
```

Comment:
1. Sorry about this. I'm editing file online, from github.  
2. If using just `$firstBit[0]`, and `@package` name will be empty in comment block, e.g. `@package ''`, then PHP throw notice: 
`PHP Notice:  Uninitialized string offset: 0 in .composer/vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PEAR/Sniffs/Commenting/FileCommentSniff.php on line 350`,
because `$firstBit[0]` is not set, but used.
3. Fixed.